### DISCUSSION TITLE
Label Value styling

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#528] [General] [answered]
Do we have any standards on how we should style labels and values in our pages? I've attached an example use case below. Should the label be bold or the values be bold? 
![image](https://user-images.githubusercontent.com/64559618/85635479-d10afe00-b632-11ea-8821-217d977aaf53.png)

**Comment by richbachman** [comment, marked as answer]
Hi @pguptaDS. We suggest using semiBold text for the labels. 

You can find the label Sketch symbol by going to forms > symbol parts > Console > label. Or you can use the "color-text (semibold)"  text style in the "console theme" library.

Also note we do have a Description List component slated for our 2020.11.03 release: https://paste.twilio.design/roadmap

**Reply by serifluous** [reply]
If you create the text style yourself, you'll have to use Whitney Medium because console doesn't load Whitney Semibold.

**Comment by pguptaDS** [comment]
So I can't use this right? Just wanted to confirm. 
![image](https://user-images.githubusercontent.com/64559618/85639875-fd2c7c00-b63e-11ea-90a8-f3eaf7a9d894.png)

**Reply by serifluous** [reply]
You can use that. It gives you Whitney medium
